<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Account</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	background-color: #B8E0FF;
}

QWidget {
	background-color: #B8E0FF;
}

QPushButton {
    border-radius: 0px;
    border: 2px solid #0078d4;
    background-color: #0078d4;
    color: #ffffff;
    padding: 3px 14px;
}

QPushButton:hover {
    background-color: #ffffff;
	color: #0078d4;
}

QPushButton:pressed {
    background-color: #ffffff;
	color: #ffffff;
}

QLineEdit {
	border-radius: 10px;
	padding: 10px 14px;
	margin: 2px;
	background-color: #ffffff;
}

QCheckBox::indicator {
	border-radius: 5px;
}

QCheckBox::indicator:unchecked {
    background-color: #ffffff;
    border: 2px solid #0078d4;
}

QCheckBox::indicator:checked {
    background-color: #0078d4;
    border: 2px solid #0078d4;
}

QRadioButton::indicator {
	border-radius: 8px;
	border: 2px solid #0078d4;
}

QRadioButton::indicator:unchecked {
    background-color: #ffffff;
}

QRadioButton::indicator:checked {
    background-color: #0078d4;
}

QComboBox {
	background-color: #ffffff;
}

QComboBox QAbstractItemView {
	background-color: #ffffff;
}

QComboBox QAbstractItemView::indicator {
	border-radius: 5px;
}

QComboBox QAbstractItemView::indicator:unchecked {
    background-color: #ffffff;
    border: 2px solid #000000;
}

QComboBox QAbstractItemView::indicator:checked {
    background-color: #000000;
    border: 2px solid #000000;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Quicksand</family>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:700;&quot;&gt;Personal Information&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nameLine">
        <property name="placeholderText">
         <string>Full Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="positionLine">
        <property name="placeholderText">
         <string>Position</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Policy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>25</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Quicksand</family>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:700;&quot;&gt;Account Information&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="usernameLine">
        <property name="placeholderText">
         <string>Username</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="passwordLine">
        <property name="echoMode">
         <enum>QLineEdit::EchoMode::Password</enum>
        </property>
        <property name="placeholderText">
         <string>Password</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="confirmPassLine">
        <property name="echoMode">
         <enum>QLineEdit::EchoMode::Password</enum>
        </property>
        <property name="placeholderText">
         <string>Confirm Password</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="cancelBttn">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="createBttn">
        <property name="text">
         <string>Create</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>473</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
